client/src/api/EdenAPI.ts

```diff
@@ -0,0 +1,80 @@
+import axios from 'axios';
+import { ReceiptData } from '../interfaces/interfaces';
+
+/**
+ * Convert date from DD/MM/YYYY to YYYY/MM/DD
+ *
+ * This is needed because the API returns the date in DD/MM/YYYY format,
+ * but our React code expects the date in YYYY/MM/DD format.
+ *
+ * @param dateString Date in DD/MM/YYYY format
+ * @returns Date in YYYY/MM/DD format
+ */
+function convertDDMMYYYYToYYYYMMDD(dateString: string) {
+  const parts = dateString.split('/');
+
+  // Extracting the day, month, and year from the original date
+  let day = parts[0];
+  let month = parts[1];
+  let year = parts[2];
+
+  // Assuming the year is in YY format and needs to be converted to YYYY
+  if (year.length === 2) {
+    year = parseInt(year, 10) < 50 ? '20' + year : '19' + year;
```

Comment:
Not that I think we will have any reciepts from before 1950 or after 2050, but why choose 50 to be the threshold as 19XX, 20XX. (Because it works is good enough lol)

Reply:
Somewhat arbitrary- it just works